frontend/src/app/main/ui/shapes/custom_stroke.cljs

```diff
@@ -327,7 +327,7 @@
                          (obj/clone))
 
         props        (cond-> props
-                       (d/not-empty? (:shadow shape))
+                       (and (d/not-empty? (:shadow shape)) (not= :frame (:type shape)))
```

Comment:
I think we should use here the `app.common.page.helpers/frame-shape?` predicate.